src/cnn_feature_extractor/README.txt

```diff
@@ -0,0 +1,18 @@
+
+Using Caffe (http://caffe.berkeleyvision.org/), cnn_feature_extractor extracts a feature vector from each input image. It is assumed Caffe has been built and the header files and library (libcaffe.so) are available.
```

Comment:
We might want to modify this line and say that we require Caffe to by build with CMake, since the CMake-generated `CaffeConfig.cmake` file is what will supply the header files and such.